libs/deso-protocol/src/index.ts

```diff
@@ -14,11 +14,14 @@ import { Tutorial } from './lib/tutorial/Tutorial';
 import { Wallet } from './lib/wallet/Wallet';
 import { Transactions } from './lib/transaction/Transaction';
 export class Deso {
-  constructor() {
+  constructor(nodeUri?: string, identityConfig?: IdentityConfig) {
```

Comment:
See my comment above on adding a DesoConfig here, accidently wrote it for the dist file